gammapy/estimators/flux_point.py

```diff
@@ -771,16 +769,13 @@ def __init__(
         self.norm_values = norm_values
         self.n_sigma = n_sigma
         self.n_sigma_ul = n_sigma_ul
-        self.backend = backend
-        if optimize_opts is None:
-            optimize_opts = {}
-        if covariance_opts is None:
-            covariance_opts = {}
-        self.optimize_opts = optimize_opts
-        self.covariance_opts = covariance_opts
-        self.reoptimize = reoptimize
         self.selection_optional = selection_optional
 
+        if fit is None:
+            fit = Fit(reoptimize=False, confidence_opts={"backend": "scipy"})
```

Comment:
This change solves a real issue: https://docs.gammapy.org/dev/tutorials/analysis/1D/spectral_analysis.html#Compute-Flux-Points In the current dev version the UL of the last flux point in the link above is not computed correctly. This is partly a know issue, as `iminuit` showed convergence problems for "linear" fit statistics. The `scipy` backend  with the Brentq algorithm behaves much more stable in this regard. However I'm not sure what to do about this in general...